src/test/fuzz/net_permissions.cpp

```diff
@@ -23,6 +23,7 @@ void test_one_input(const std::vector<uint8_t>& buffer)
                                                                                              NetPermissionFlags::PF_FORCERELAY,
                                                                                              NetPermissionFlags::PF_NOBAN,
                                                                                              NetPermissionFlags::PF_MEMPOOL,
+                                                                                             NetPermissionFlags::PF_ADDR,
```

Comment:
This change can be squashed into commit _Add whitelist flag enabling non-cached addr sharing_